### PR TITLE
fix: type referencing styled components

### PR DESCRIPTION
**Pull request by momentiris**
**Ticket:** [#97](https://trello.com/c/8IBnAhgp/97-generic-components-type-typos)
**Intent:**
Use declared types correctly in generic components that uses styled-system

### All of the following commands should pass.

- [x] `npm test`
- [x] `npm run lint`
- [ ] `npm run cypress`

### Browser testing

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] IE11
- [ ] Edge

### CODE REVIEW

File: cypress/support/step_definitions/start.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
 Given('I open the startpage', () => {
```
Comment (lundeeell) [inline cypress/support/step_definitions/start.js]: Is login startpage?

File: src/components/ExperienceList/ExperienceList.tsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React from 'react'
-import styled from '@emotion/styled'
 import { Taxonomy } from '../../generated/myskills'
+import List from '../List'
 
 type MutaionVariables = {
```
Comment (lessp) [inline src/components/ExperienceList/ExperienceList.tsx]: Typo!

File: src/components/ExperienceList/ExperienceList.tsx [hunk 3]
```diff
@@ -13,23 +13,6 @@ type MutaionVariables = {
...
-
-    &:hover {
-      transform: scale(1.1);
-    }
-  }
-`
-
 interface ExperienceListProps {
   list: Taxonomy.Result[]
   addExperience?: (variables: MutaionVariables) => void
```
Comment (lessp) [inline src/components/ExperienceList/ExperienceList.tsx]: Same here 🙂

File: src/components/ExperienceList/ExperienceList.tsx (hunk 2 above)
Comment (momentiris) [reply src/components/ExperienceList/ExperienceList.tsx]: Fixed!

File: src/components/ExperienceList/ExperienceList.tsx (hunk 3 above)
Comment (momentiris) [reply src/components/ExperienceList/ExperienceList.tsx]: Fixed!

File: cypress/support/step_definitions/start.js (hunk 1 above)
Comment (momentiris) [reply cypress/support/step_definitions/start.js]: Nope! Fixed that typo. Though it seems to me that this test is not working properly but I judge it as being outside the scope of this PR. Also I have no experience with cypress.

**Pull request merged**